services/ml/compilation_delegate_dnnl.cc

```diff
@@ -1229,6 +1231,63 @@ int32_t CompilationDelegateDnnl::AddSoftmax(
   return mojom::NOT_ERROR;
 }
 
+int32_t CompilationDelegateDnnl::AddLogistic(
```

Comment:
Could you check whether can consolidate code with `AddFusedActivation`? I suppose both of them are using `dnnl_eltwise` primitive, correct?

Reply:
Yes, I will try it.